bookshelf/Shelf 1 Getting Started/overview/src/README.md

```diff
@@ -0,0 +1,33 @@
+# Open Device Partnership Overview
+![Open Device Partnership](media/odp.png)
+
+## Overview
+An alliance of industry-leading PC ecosystem partners promoting secure, reusable, and trusted system software for client devices
+
+The Open Device Partnership (ODP) is an open-source initiative focused on:
+- enhancing device security
+- simplifying cross-architecture development (ARM & x86 standards)
+- strengthening fundamentals (“raise all boats”)
+- accelerating the delivery of high-quality devices
+
+Designed from the onset to be inclusive of all device ecosystem partners and stakeholders
+- one stop location for everything needed to build a Windows client device
+
+Optimized for Windows devices but only successful if Linux can also thrive
```

Comment:
Maybe we want avoid being specific here? How about "but only successful if other Operating Systems can also thrive"?